app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailFragment.kt

```diff
@@ -240,10 +262,77 @@ class AgendaItemDetailFragment : Fragment(R.layout.fragment_agenda_item_detail)
             descriptionTextView.isClickable = isEditing
 
             time.isClickable = isEditing
+            timeButton.isVisible = isEditing
+            timeButton.isClickable=isEditing
```

Comment:
I'm not sure if `isClickable` has an influence.. I think this refers to showing a ripple effect. Better use `timeButton.isEnabled = isEditing` to disable/enable it